1266/en.md

```diff
@@ -0,0 +1,141 @@
+### 1266 - Points in Rectangle
+
+**Prolblem Description:** This problem is about finding the number of points in a rectangle whose sides are parallel to axis. All the points and rectangles consist of 2D Cartesian co-ordinates. A point that lies in the boundary of a rectangle is considered inside.
+
+**Observation:** The Idea was quite simple that after seeing a large number of query over a grid that somehow we have to do a little bit storing of precalculated result of the previous points.
+Here We are using a well known <em>technique Bit Index Tree or Fenwick Tree over 2-d Grid</em> If you do not know the algo visit the it and its varaints here
+<a here="https://cp-algorithms.com/data_structures/fenwick.html" target="_blank">
```

Comment:
The anchor tag is incorrect. The link is not showing up in the paragraph. It should be `href` instead of `here`.